src/vs/workbench/contrib/chat/common/promptSyntax/service/promptsService.ts

```diff
@@ -209,7 +209,7 @@ export class PromptsService extends Disposable implements IPromptsService {
 			}
 		}
 
-		return result;
+		return [...new Set(result)];
```

Comment:
I doubt this will work - the `URI` object isn't a singleton, e.g `URI.parse('file:///aaa.bbb.txt') !== URI.parse('file:///aaa.bbb.txt')` and the JS set does work by object identity